site/src/pages/release-notes/1.13.3.mdx

```diff
@@ -0,0 +1,53 @@
+---
+date: 2021-11-23
+---
+
+## 🌟 New features
+
+- N/A
+
+## 📈 Improvements
+
+- N/A
+
+## 🛠️ Bug fixes
+
+- <type://DecodingClient> does not raise an exception anymore when duplicated encodings are specified. #3932 #3934
+  - The duplicated encodings are silently ignored.
+- `unfinishedResponse` is correctly decremented when the <type://HttpResponse> is cancelled. #3920
+- The text in the description sections are now correctly rendered in DocService. #3917
+- The status of a protocol violation is now correctly logged by <type://LoggingService> #3914
+    - A response headers is exactly written once when a protocol violation error is raised.
+- `GrpcDocStringExtractor` now reads description from `bin`, `desc`, `dsc`, `pb`, and `protobin` file. #3912
+- You no longer see `Maximum active streams violated for this endpoint.` exception anymore. #3858 #3908
+- <type://EurekaEndpointGroupBuilder> now fails if an empty string is specified for `appName` and
+  other parameters. #3903
+- OkHttp can be used with self-signed certificate in Armeria. #3902
+  - Users can use `https://127.0.0.1:` with the default self-signed certificate.
+- <type://GraphqlService> now produces `graphql+json` response when the `accept` header is `*`. #3898
+
+## 🏚️ Deprecations
+
+- N/A
+
+## ☢️ Breaking changes
+
+- N/A
+
+## ⛓ Dependencies
+
+- N/A
+
+## 🙇 Thank you
+
+<ThankYou usernames={[
+  'anuraaga',
+  'eisig',
+  'heowc',
+  'hyangtack',
+  'ikhoon',
+  'jrhee17',
+  'minwoox',
+  'ohadgur',
```

Comment:
can you also add techno who initially reported #3858?

Reply:
Oops, I didn't notice that. Thanks! 😉

Reply:
That is a hole in the release note generation script. 🙈

Reply:
Yeah, I should've double-checked. 😆 
But the script is really great because it reduces the time to create the release note.